src/libhirte/common/cfg.c

```diff
@@ -102,7 +102,7 @@ int cfg_load_complete_configuration(
                 result = cfg_load_from_file(config, default_config_file);
                 if (result < 0) {
                         fprintf(stderr,
-                                "Error loading default configuration file '%s': '%s'.\n",
+                                "Error: loading default configuration file '%s' failed: '%s'.\n",
```

Comment:
Wouldn't this change make the logs info too duplicated?

```
<DATE>  Error: loading default configuration file '<FILE>' failed: '<ERROR MESSAGE>'
```

Isn't the original more readable?

```
<DATE>  Error loading default configuration file '<FILE>': '<ERROR MESSAGE>'
```

Reply:
No, on any error here the daemon will stop. The print won't stop the daemon, but by returning the `result` one line below we will - it will bubble up all the way to main and then `return EXIT_FAILURE`.